src/inverseRobot.js

```diff
@@ -25,6 +25,23 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const values = Object.values(robot);
+
+  for (let i = 0; i < values.length; i++) {
+    for (let j = i + 1; j < values.length; j++) {
+      if (values[i] === values[j]) {
+        return null;
+      }
+    }
+  }
```

Comment:
let's use array methods to find repeated or unique values instead of 2 nested cycles